code_challenges.rb

```diff
@@ -9,7 +9,25 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+#PROCESS:
+    #create a method - letter_select
+    #parameters - array, letter
+    #set a new empty array - newArray
+    #use .each method and pass in a value
+    #if value include? given letter, print value
+    #inputs & outputs - ['coffee', 'tea', 'juice', 'water', 'soda water'] , 'o' --> ['coffee', 'soda water'] , 't' ---> ['tea', 'water', 'soda water']
 
+def letter_select (array, letter)
+    newArray = []
+    array.each do |value|
```

Comment:
Oh cool! When I ran this it gave me three different returns 
["tea"]
["tea", "water"]
["tea", "water", "soda water"]

This is the difference between each and map. Great job though! It's nice to see the differences in action